Management/src/main/java/com/stack/jspdemo/model/Product.java

```diff
@@ -0,0 +1,64 @@
+package com.stack.jspdemo.model;
+
+import javax.validation.constraints.Size;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class Product {
```

Comment:
Why not making `Product` a `final` class?

Reply:
Yes of course but I really don't know what would be the technical reason for that, can explain me why?